index.d.ts

```diff
@@ -1,3 +1,7 @@
+interface Options {
+	noOverlap?: boolean;
```

Comment:
You need to document this option.

See: https://github.com/sindresorhus/typescript-definition-style-guide